XIPs/xip-0-purpose-process.md

```diff
@@ -38,13 +38,19 @@ An XIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## XIP Workflow
 
-### Shepherding an XIP
+## Getting started
+
+It's important to gather feedback about your proposal or new idea within the community before advancing it further in the process. This helps to vet your idea before going through the formality of the Improvement Proposal process.
+
+Engaging with the XMTP community first will help determine the originality of an idea, and will help to avoid wasting time on something that may be rejected based on prior research.
+
+**It's recommended to create a new topic in the [XIP Ideas forum](https://github.com/orgs/xmtp/discussions/categories/xip-ideas) to do this.**
 
-Parties involved in the process are you, the champion or *XIP author*, the [*XIP editors*](#XIP-editors), and the *XMTP Core Developers* (currently the XMTP Labs team).
+### Shepherding an XIP
 
-Before you begin writing a formal XIP, you should vet your idea. Ask the XMTP community first if an idea is original to avoid wasting time on something that will be rejected based on prior research. It is thus recommended to create a new topic in the [XIPs discussion forum](https://github.com/orgs/xmtp/discussions/categories/xips) to do this.
+Parties involved in the process are you, the champion or *XIP author*, the [*XIP editors*](#xip-editors), and the *XMTP Core Developers* (currently the XMTP Labs team).
 
-Once the idea has been vetted, your next responsibility will be to present (by means of an XIP) the idea to the reviewers and all interested parties, invite editors, developers, and the community to give feedback on the aforementioned channels. You should try and gauge whether the interest in your XIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. Negative community feedback will be taken into consideration and may prevent your XIP from moving past the Draft stage.
+Once the idea has been vetted (as described in [*Getting Started*](#getting-started)), your next responsibility will be to present (by means of an XIP Draft) the idea to the reviewers and all interested parties, invite editors, developers, and the community to give feedback on the aforementioned channels. You should try and gauge whether the interest in your XIP is commensurate with both the work involved in implementing it and how many parties will have to conform to it. Negative community feedback will be taken into consideration and may prevent your XIP from moving past the Draft stage.
```

Comment:
@saulmc Here's a stab:
```suggestion
Vetting your idea within a discussion is a critical part of the XIP process. We encourage sharing the discussion with reviewers, interested parties, editors, developers, and the community to gather feedback.

Additional vetting criteria may include reviewing reference implementations or assessing whether the idea is too specific for broad adoption within the XMTP ecosystem. It’s important to gauge whether the interest in your XIP is commensurate with the effort required to implement it and how many parties will need to conform to it. While the volume of interest by way of discussion is not an explicitly disqualifying characteristic, it may play a role in advancing beyond the idea stage.

After vetting the idea as described above, your next responsibility is to present the idea as a formal XIP Draft by opening a [Pull Request](https://github.com/xmtp/XIPs/pulls).  Following this step, the [XIP Editors](#xip-editors) will review the Draft and make recommendations on its advancement.
```